frontend/src/common/ContentTile.tsx

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from "styled-components";
+
+export const colourLookup: { [key: string]: string } = {
```

Comment:
can you move this to a separate `.ts` file. `tsx` files should only export components for fast refreshing to be useful.